packages/fx-core/src/plugins/resource/bot/appStudio/appStudio.ts

```diff
@@ -200,17 +212,23 @@ export class AppStudio {
   public static async updateMessageEndpoint(
     accessToken: string,
     botId: string,
-    registration: IBotRegistration
+    endpoint: string
   ): Promise<void> {
     const axiosInstance = AppStudio.newAxiosInstance(accessToken);
 
+    const botReg = await AppStudio.getBotRegistration(accessToken, botId);
```

Comment:
Same issue here

Reply:
Good Catch! 
We may choose to pass `axiosInstance` into `AppStudio.getBotRegistration` to avoid this kind of duplication, but I prefer not to do it because it will break the consistency of these AppStudio api - I mean, each AppStudio API here, for example, `updateMessageEndpoint` got a unify interface `(accessToken, <necessary params>)`.
I prefer to keep it as it is although it has this kind of duplication.